.github/workflows/openapi-update-generate-new-sdk.yml

```
@@ -35,11 +35,34 @@ jobs:
         run: |
           current_version=$(xmlstarlet sel -t -v "//*[local-name()='artifactVersion']" pom.xml)
           echo "Current version: $current_version"
-          IFS='.-' read -r -a parts <<< "$current_version"
-          new_patch=$((parts[2] + 1))
-          new_version="${parts[0]}.${parts[1]}.$new_patch-${parts[3]}"
+
+          unique_id=$(date +%Y%m%d%H%M%S)
```

As proposed in our discussion internally, it would make more sens if this unique ID representing the latest commit short hash

A suggestion to get it :)
```suggestion
          unique_id=$(git rev-parse --short HEAD)
```

It will produce something like that: `6d1724e`

---

Got it, thanks a lot : )